unstructured/documents/html.py

```diff
@@ -9,7 +9,7 @@
 
 from lxml import etree
 
-from unstructured.logger import get_logger
+from unstructured.logger import logger
```

Comment:
Do you know why these files are showing up in your diffs? They look the same as what's on `main`